catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/RemoveCommand.java

```diff
@@ -67,69 +73,74 @@ protected Object executeWithSubject() throws Exception {
     }
 
     private Object executeRemoveFromCache() throws Exception {
-        String[] idsArray = new String[ids.size()];
-        idsArray = ids.toArray(idsArray);
-        getCacheProxy().removeById(idsArray);
-
-        List idsList = Arrays.asList(ids);
-
-        printSuccessMessage(idsList + " successfully removed from cache.");
-
-        LOGGER.info(idsList + " removed from cache using catalog:remove command");
-
+        if (CollectionUtils.isNotEmpty(ids)) {
+            getCacheProxy().removeById(ids.toArray(new String[ids.size()]));
+            printSuccessMessage(ids + " successfully removed from cache");
+            LOGGER.info(ids + " removed from cache using catalog:remove command");
```

Comment:
✏️  you could use slf4j parameterized logging for `ids` on this logger statement: https://www.slf4j.org/faq.html#logging_performance